src/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+#
+# This file is part of RGBDS.
+#
+# Copyright (c) 2020 RGBDS contributors.
+#
+# SPDX-License-Identifier: MIT
+#
+
+set(common_src
+    "extern/err.c"
+    "extern/getopt.c"
+    "version.c"
+    )
+
+if(BISON_FOUND)
+  BISON_TARGET(ASMy "asm/asmy.y"
+               "${PROJECT_SOURCE_DIR}/src/asm/asmy.c"
+               DEFINES_FILE "${PROJECT_SOURCE_DIR}/src/asm/asmy.h"
+               )
+  set(ASMy_SOURCE "${BISON_ASMy_OUTPUT_SOURCE}")
+else()
```

Comment:
I would remove the checks for the generated files. The tool is a dependency, and you can't build rbgds without it. Simply fail here instead of checking.

Reply:
Done.